src/cli.js

```diff
@@ -8,6 +10,10 @@ const run = () => {
 	const argv = yargs
 		.commandDir('commands') // commands are in the './commands/' dir
 		.demandCommand() // require a command to be used - no default behavior
+		.option('config', {
```

Comment:
I don't think we should use a command line arg parser to exclusively read from a config file - particularly because this will load `config` as an object, and there's no clear way to pass an object as a command line argument to `--config`.

Two considerations

1. Do we want to be able to override this config with command line args? Is there a use case for that? If not, we can just write a simple module to read from the config and export a plain object. If there is a use case for having this assignable from the command line, break it up into separate options, e.g. `.option('subdomain')` and `option('tx-project')` so that they can be individually documented and easily passed in as CLI args.
2. `config` is a vague name, since every CLI option is effectively a 'configuration' rule. If you keep a reference to the whole package.config object, I'd go with something like `packageConfig` to make it more clear what it it's coming from.